package.json

```diff
@@ -58,4 +58,4 @@
     "prettier": "^2.2.1"
   },
   "proxy": "http://localhost:5000"
-}
+}
```

Comment:
new line !!

Reply:
i added a new line but when i saved the file it was removed